mx.truffleruby/mx_truffleruby.py

```diff
@@ -202,9 +202,10 @@ def ruby_testdownstream_sulong(args):
     jt('test', 'cexts')
     jt('test', 'specs', ':capi')
     jt('test', 'specs', ':library_cext')
+    jt('test', 'mri', '--cext')
     jt('test', 'mri', '--openssl')
     jt('test', 'mri', '--syslog')
-    jt('test', 'mri', '--cext')
+    jt('test', 'mri', 'zlib')
```

Comment:
why are other options and zlib is not?

Reply:
Because it's a single test file, so this is enough.
But maybe it's good to accept `--zlib` for consistency.

Reply:
ok then